redisgraph/query_result.py

```diff
@@ -0,0 +1,74 @@
+from prettytable import PrettyTable
+
+
+class QueryResult(object):
+
+    LABELS_ADDED = 'Labels added'
+    NODES_CREATED = 'Nodes created'
+    PROPERTIES_SET = 'Properties set'
+    RELATIONSHIPS_CREATED = 'Relationships created'
+    INTERNAL_EXECUTION_TIME = 'internal execution time'
+
+    def __init__(self, result_set, statistics):
+        self.result_set = result_set
+        (self.statistics, self.parsed_statistics) = \
+            self._retrieve_data_from_statistics(statistics)
+
+
+    """Prints the data from the query response:
+       1. First row result_set contains the columns names. Thus the first row in PrettyTable
+          will contain the columns.
+       2. The row after that will contain the data returned, or 'No Data returned' if there is none.
+       3. Prints the statistics of the query.
+    """
+    def pretty_print(self):
+        tbl = PrettyTable(self.result_set[0])
+        for row in self.result_set[1:]:
+            tbl.add_row(row)
+
+        if len(self.result_set) == 1:
+            tbl.add_row(['No data returned.'])
+
+        print(str(tbl) + '\n')
+
+        for stat in self.statistics:
+            print(stat)
+
+    def _retrieve_data_from_statistics(self, statistics):
+        parsed_statistics = {
```

Comment:
Wouldn't you prefer to have a class variable foreach statistic?
e.g. self.labels_added = self._get_value(self.LABELS_ADDED)